content/blog/posts/tji-travels-under-the-pink-dome.md

```diff
@@ -0,0 +1,28 @@
+---
+title: TJI Travels Under the Pink Dome
+subtitle: ""
+date: 2021-05-10
+authors:
+  - Eva Ruth Moravec
+---
+Every other year, the pink granite building that sits in the heart of Austin buzzes with activity for six months as lawmakers gather to debate, consider, toss out and approve legislation that governs life in Texas. 
```

Comment:
Picture of the building? 😝 (Just a thought)

Reply:
good point. I'm not sure how to add one though – should I wait until it's in the CMS? I was hoping to use one of the [Getty embeddable images ](https://www.gettyimages.com/search/2/image?phrase=texas+capitol)

Reply:
Oh very nice. Probably @nholden might know the answer :)